.github/workflows/ci-kube-prometheus.yml

```diff
@@ -0,0 +1,133 @@
+# Copyright Broadcom, Inc. All Rights Reserved.
+# SPDX-License-Identifier: APACHE-2.0
+
+name: '[CI/CD] Special CI for kube-prometheus'
+on: # rebuild any PRs and main branch changes
+  pull_request_target:
+    types:
+      - opened
+      - reopened
+      - synchronize
+    branches:
+      - main
+      - bitnami:main
+    paths:
+      - 'bitnami/kube-prometheus/**'
+# Remove all permissions by default
+permissions: {}
+# Avoid concurrency over the same PR
+concurrency:
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+jobs:
+  update-prometheus-rules:
+    runs-on: ubuntu-latest
+    name: Automatically update default Prometheus rules based on upstream
+    permissions:
+      contents: write
```

Comment:
Not really needed. We are using the BITNAMI_BOT_TOKEN below to commit the changes, so we don't need to add permissions to the GITHUB_TOKEN

Reply:
Copy-pasted from CI workflow, I can remove it if needed